.github/workflows/coverage.yml

```diff
@@ -14,33 +14,21 @@ jobs:
         os: [ ubuntu-latest ]
 
     steps:
-      - name: Git Checkout
-        uses: actions/checkout@v3
+      - name: Git checkout
+        uses: actions/checkout@v4
         with:
           submodules: recursive
-      - name: Rust Toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
         with:
-          profile: minimal
-          toolchain: stable
           # We need this component to generate source coverage in stable
           components: llvm-tools-preview
-          override: true
-      # Wrapper for running coverage with -C instrument-coverage
-      - name: Cargo Install Coverage Wrapper
-        uses: actions-rs/cargo@v1
```

Comment:
🗺️ The `actions-rs/cargo` action was a pretty thin layer around simple `cargo [subcommand]` operations. I'm not sure how much value it was really adding.